subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -106,9 +106,12 @@ static void stopped(void)
 static void dfu_target_callback_handler(enum dfu_target_evt_id evt)
 {
 	switch (evt) {
-	case DFU_TARGET_EVT_TIMEOUT:
+	case DFU_TARGET_EVT_ERASE_PENDING:
 		send_evt(FOTA_DOWNLOAD_EVT_ERASE_PENDING);
 		break;
+	case DFU_TARGET_EVT_TIMEOUT:
+		send_evt(FOTA_DOWNLOAD_EVT_ERASE_TIMEOUT);
```

Comment:
Does this event need to be handled in other places as well?

Reply:
Yes. Some of the places that process FOTA_DOWNLOAD_EVT_ERASE_PENDING treat non-mapped events as unknown and report errors on those.

Reply:
@tomi-font: This should be handled.

Reply:
@MarkusLassila Can you re-review?

Reply:
Apologies for delay. It seems that I am not receiving all the notifications from these.

@tomi-font
This will write an error to the log, if the event is received. It would be better to just process it, or at least add processing so that the people responsible for these could review this.

https://github.com/nrfconnect/sdk-nrf/blob/7c10920b971b1ef3285f8b86703c5430de8556d9/samples/cellular/modem_shell/src/fota/fota.c#L81C4-L105

And if I am not mistaken, the same happens with these:
https://github.com/nrfconnect/sdk-nrf/blob/7c10920b971b1ef3285f8b86703c5430de8556d9/subsys/net/lib/aws_fota/src/aws_fota.c#L733C1-L800C1

https://github.com/nrfconnect/sdk-nrf/blob/7c10920b971b1ef3285f8b86703c5430de8556d9/subsys/net/lib/azure_fota/azure_fota.c#L228C1-L268C3